pop_block/index.js

```diff
@@ -0,0 +1,271 @@
+    ['GYD', false], //Baku, AZ (ME)
+    ['HAM', true], //Hamburg, DE (EEUR)
+    ['HAN', false], //Hanoi, VN (APAC)
+    ['HEL', true], //Helsinki, FI (EEUR)
+    ['HGH', false], //Hangzhou, CN (CHINA)
+    ['HKG', false], //Hong Kong, HK (APAC)
+    ['HNL', false], //Honolulu, HI, US (WNAM)
+    ['HNY', false], //Hengyang, CN (CHINA)
+    ['HRE', false], //Harare, ZW (AFR)
+    ['HYD', false], //Hyderabad, IN (APAC)
+    ['IAD', false], //Ashburn, VA, US (ENAM)
+    ['IAH', false], //Houston, TX, US (ENAM)
+    ['ICN', false], //Seoul, KR (APAC)
+    ['IND', false], //Indianapolis, IN, US (ENAM)
+    ['ISB', false], //Islamabad, PK (APAC)
+    ['IST', true], //Istanbul, TR (EEUR)
+    ['JAX', false], //Jacksonville, FL, US (ENAM)
+    ['JHB', false], //Johor Bahru, MY (APAC)
+    ['JIB', false], //Djibouti, DJ (AFR)
+    ['JNB', false], //Johannesburg, ZA (AFR)
+    ['JSR', false], //Jashore, BD (APAC)
+    ['JXG', false], //Jiaxing, CN (CHINA)
+    ['KBP', true], //Kyiv, UA (EEUR)
+    ['KEF', true], //Reykjavík, IS (WEUR)
+    ['KGL', false], //Kigali, RW (AFR)
+    ['KHI', false], //Karachi, PK (APAC)
+    ['KIV', true], //Chișinău, MD (EEUR)
+    ['KIX', false], //Osaka, JP (APAC)
+    ['KTM', false], //Kathmandu, NP (APAC)
+    ['KUL', false], //Kuala Lumpur, MY (APAC)
+    ['KWI', false], //Kuwait City, KW (ME)
+    ['LAD', false], //Luanda, AO (AFR)
+    ['LAS', false], //Las Vegas, NV, US (WNAM)
+    ['LAX', false], //Los Angeles, CA, US (WNAM)
+    ['LCA', true], //Nicosia, CY (EEUR)
+    ['LED', true], //Saint Petersburg, RU (EEUR)
+    ['LHE', false], //Lahore, PK (APAC)
+    ['LHR', true], //London, GB (WEUR)
+    ['LHW', false], //Lanzhou, CN (CHINA)
+    ['LIM', false], //Lima, PE (SAM)
+    ['LIS', true], //Lisbon, PT (WEUR)
+    ['LOS', false], //Lagos, NG (AFR)
+    ['LUX', true], //Luxembourg City, LU (WEUR)
+    ['LYA', false], //Luoyang, CN (CHINA)
+    ['MAA', false], //Chennai, IN (APAC)
+    ['MAD', true], //Madrid, ES (WEUR)
+    ['MAN', true], //Manchester, GB (WEUR)
+    ['MBA', false], //Mombasa, KE (AFR)
+    ['MCI', false], //Kansas City, MO, US (ENAM)
+    ['MCT', false], //Muscat, OM (ME)
+    ['MDE', false], //Medellin, CO (SAM)
+    ['MEL', false], //Melbourne, AU (OC)
+    ['MEM', false], //Memphis, TN, US (ENAM)
+    ['MEX', false], //Mexico City, MX (WNAM)
+    ['MFE', false], //Mcallen, TX, US (ENAM)
+    ['MFM', false], //Macau, MO (APAC)
+    ['MGM', false], //Montgomery, AL, US (ENAM)
+    ['MIA', false], //Miami, FL, US (ENAM)
+    ['MLE', false], //Male, MV (APAC)
+    ['MNL', false], //Manila, PH (APAC)
+    ['MPM', false], //Maputo, MZ (AFR)
+    ['MRS', true], //Marseille, FR (WEUR)
+    ['MRU', false], //Port Louis, MU (AFR)
+    ['MSP', false], //Minneapolis, MN, US (ENAM)
+    ['MUC', true], //Munich, DE (WEUR)
+    ['MXP', true], //Milan, IT (EEUR)
+    ['NAG', false], //Nagpur, IN (APAC)
+    ['NAY', false], //Langfang, CN (CHINA)
+    ['NBG', false], //Ningbo, CN (CHINA)
```

Comment:
There is an error with NBG it should be NGB, it is an error on cloudflare status page.
Please correct, and remove the conflict with with README.

Then it is ready to merge. Thanks